services/github/github-tag.tester.js

```diff
@@ -28,6 +28,10 @@ t.create('Tag (repo not found)')
   .get('/v/tag/badges/helmets.json')
   .expectBadge({ label: 'tag', message: 'repo not found' })
 
+t.create('Tag (filter by prefix')
+  .get('/v/tag/ros/rosdistro.json?subpackage=galactic')
+  .expectBadge({ label: 'tag', message: isKebabYearMonthDay })
```

Comment:
You don't need to define a custom regex validator for this. `Joi.date().required()` will do what you're trying to do here.

Reply:
That worked. Thanks!